docs/reference/aggregations/bucket/range-field-note.asciidoc

```diff
@@ -136,35 +137,35 @@ calculated over the ranges of all matching documents.
     "november_data" : {
       "buckets" : [
         {
-          "key" : 1572220800000,
```

Comment:
I think the doc tests are failing because date_histo will show both the key and key_as_string.  E.g. 

```
                {
                    "key_as_string": "2015-01-01",
                    "key": 1420070400000,
                    "doc_count": 3
                },
```